docs/about_Puppetization.md

```diff
@@ -0,0 +1,49 @@
+# Puppetization
+## about_Puppetization
+
+# SHORT DESCRIPTION
+
+Explanation of how puppet.dsc turns DSC resources into Puppet Types & Providers
```

Comment:
Is `puppet.dsc` the name of the builder?

Reply:
It's the name of the PowerShell module.